### PR TITLE
Add ntpservers param

### PR DESCRIPTION
**Pull request by brandonweeks**
Add ntpservers param with the same format as puppetlabs-dhcp.

### CODE REVIEW

File: templates/dhcpd.conf.erb [hunk 1]
```diff
@@ -76,6 +82,7 @@ next-server <%= @pxeserver %>;
 filename "<%= @pxefilename %>";
 <% end -%>
 
+
```
Comment (mmoll) [inline templates/dhcpd.conf.erb]: superfluous extra newline here

File: templates/dhcpd.conf.erb [hunk 2]
```diff
@@ -47,6 +47,12 @@ option domain-name "<%= @dnsdomain.first %>";
 <% end -%>
 option domain-name-servers <%= @nameservers.join( ', ') %>;
```
Comment (mmoll) [inline templates/dhcpd.conf.erb]: maybe also delete that newline

**Comment by mmoll** [comment]
normally I'd say "use the options hash", but I like parameter compatibily :+1:

**Comment by brandonweeks** [comment]
@mmoll whitespace fixed.

**Comment by domcleal** [comment]
Thanks @brandonweeks!

**Pull request merged**